### PR TITLE
Remove push module from AppCenter

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] ~Did you add unit tests if this modifies the Windows code?~
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

- Remove modules + targets + bindings
- Remove UI from test apps
- Remove tests if any
- Make sure that in core there are no code related to modules 
- Edit build.cake
- Edit codeql.yml
- Remove nuspec
- Remove send-push script
- Edit ac-build-config

## Related PRs or issues

[AB#83309](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/83309)

### CODE REVIEW

**Review by vvechkanov — APPROVED** [review]

File: Apps/Contoso.Android.Puppet/MainActivity.cs [hunk 1]
```diff
@@ -26,7 +25,6 @@ public class MainActivity : AppCompatActivity
         protected override void OnDestroy()
```
Comment (MatkovIvan) [inline Apps/Contoso.Android.Puppet/MainActivity.cs:25]: Remove this override

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/Contoso.Forms.Puppet.Droid.csproj [hunk 2]
```diff
@@ -76,15 +76,6 @@
...
-    <Reference Include="Xamarin.Firebase.Common">
-      <HintPath>..\..\..\packages\Xamarin.Firebase.Common.71.1610.0\lib\monoandroid90\Xamarin.Firebase.Common.dll</HintPath>
-    </Reference>
-    <Reference Include="Xamarin.Firebase.Iid">
-      <HintPath>..\..\..\packages\Xamarin.Firebase.Iid.71.1710.0\lib\monoandroid90\Xamarin.Firebase.Iid.dll</HintPath>
-    </Reference>
-    <Reference Include="Xamarin.Firebase.Messaging">
-      <HintPath>..\..\..\packages\Xamarin.Firebase.Messaging.71.1740.0\lib\monoandroid90\Xamarin.Firebase.Messaging.dll</HintPath>
-    </Reference>
     <Reference Include="HockeySDK.AndroidBindings">
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/Contoso.Forms.Puppet.Droid.csproj:79]: HockeySDK ??!

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/Contoso.Forms.Puppet.Droid.csproj [hunk 3]
```diff
@@ -365,12 +339,7 @@
...
   <Import Project="..\..\..\packages\Xamarin.GooglePlayServices.Stats.71.1601.0\build\MonoAndroid90\Xamarin.GooglePlayServices.Stats.targets" Condition="Exists('..\..\..\packages\Xamarin.GooglePlayServices.Stats.71.1601.0\build\MonoAndroid90\Xamarin.GooglePlayServices.Stats.targets')" />
   <Import Project="..\..\..\packages\Xamarin.GooglePlayServices.Tasks.71.1601.0\build\MonoAndroid90\Xamarin.GooglePlayServices.Tasks.targets" Condition="Exists('..\..\..\packages\Xamarin.GooglePlayServices.Tasks.71.1601.0\build\MonoAndroid90\Xamarin.GooglePlayServices.Tasks.targets')" />
-  <Import Project="..\..\..\packages\Xamarin.Firebase.Common.71.1610.0\build\MonoAndroid90\Xamarin.Firebase.Common.targets" Condition="Exists('..\..\..\packages\Xamarin.Firebase.Common.71.1610.0\build\MonoAndroid90\Xamarin.Firebase.Common.targets')" />
   <Import Project="..\..\..\packages\Xamarin.GooglePlayServices.Base.71.1610.0\build\MonoAndroid90\Xamarin.GooglePlayServices.Base.targets" Condition="Exists('..\..\..\packages\Xamarin.GooglePlayServices.Base.71.1610.0\build\MonoAndroid90\Xamarin.GooglePlayServices.Base.targets')" />
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/Contoso.Forms.Puppet.Droid.csproj:344]: Do we still need it?

File: swagger/ingestion.yaml [hunk 4]
```diff
@@ -483,20 +483,6 @@ definitions:
...
-    allOf:
-      - $ref: '#/definitions/Log'
-    properties:
-      pushToken:
-        type: string
-        description: >
-          The PNS handle for this installation.
-    required:
-        - pushToken
-
```
Comment (MatkovIvan) [inline swagger/ingestion.yaml:499]: Do not touch this file

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

**Review by MatkovIvan — APPROVED** [review]

**Review by vvechkanov — APPROVED** [review]

**Pull request merged**